primeqa/tableqa/metrics/answer_accuracy.py

```diff
@@ -0,0 +1,23 @@
+import ast
+import logging
+logger = logging.getLogger(__name__)
+
+def compute_denotation_accuracy(predicted_answers,gold_answers):
```

Comment:
Why not use a pre-existing accuracy metric?

Reply:
Denotation accuracy is very specific to tableqa.